### DISCUSSION TITLE
Using the hextra button shortcode inside a Steps shortcode

### DISCUSSION THREAD

**Discussion by adamswbrown** [imfing/hextra#424] [General]
Hi All,

Im new to Hugo and Hextra, but I would like to use the Hero button inside a Steps shortcode 

this is what I get when I try to include the hero code, in the same way its used on the main page

![CleanShot 2024-07-10 at 13 23 21](https://github.com/imfing/hextra/assets/29273106/b4cc960e-5224-423e-a98e-6bf55fdd75ec)

This is the markdown I'm using

```markdown
---
date: '2024-06-23T17:24:48+01:00'
type: docs
title: 'test'
toc: false
weight: 1
--- 
 
 {{% steps %}}

### Request a Licence

**Microsoft Solution Assessment** teams have unlimited access, individual license requests are not required.  

**Field requests** for running an AVS Express assessment should come via the Solution Assessment Global Request Tool.

</br>
<div class="mb-6"> 
{{< hextra/hero-button text="Request a Licence" link="https://aka.ms/ISVForm" >}}
</div>

{{% /steps %}}
```

**Comment by chemlord** [comment]
I don't think the shortcode supports the buttons (I could be wrong), however you can use cards as an alternative and I think it might suit it better due to the colour changing and ability to allow for an icon.

{{< cards >}}
  {{< card link="../callout" title="Request A Licence" icon="cursor-click" link="https://aka.ms/ISVForm" >}}
{{< /cards >}}

You can change the icons to any of heroiconsv1: https://v1.heroicons.com/

**Comment by imfing** [comment]
I think the reason is because mixing HTML & Markdown. Basically the button HTML code was treated as Markdown.

The inner part of the `steps` are supposed to be Markdown. 

```
</br>
<div class="mb-6"> 
{{< hextra/hero-button text="Request a Licence" link="https://aka.ms/ISVForm" >}}
</div>
```
These lines will produce HTML code. Thus causing the issue you saw.

You could remove the steps shortcode, and just use plain Markdown ordered list.

**Comment by adamswbrown** [comment]
Can you also change the colour?

**Reply by imfing** [reply]
Nope, you have to either override the CSS or override the entire shortcode component. I don't recommend using this component `hextra/hero-button` since it's supposed to be internal to Hextra landing page.

You can use Tailwind CSS to create your own components: https://github.com/imfing/hextra/discussions/111

**Comment by adamswbrown** [comment]
I’d like to learn a little bit more on how I use tailwind, I don’t really
understand What are you linked to? Is there a clearer explanation?
